### DISCUSSION TITLE
Adobe Developer / React Spectrum Discord?

### DISCUSSION THREAD

**Discussion by Cuttsy27** [adobe/react-spectrum#5024] [Q&A] [answered]
Is there a Discord or Slack available for general discussion around accessibility, react-aria ideas or best practices etc or should all discussions happen on Github?

**Comment by snowystinger** [comment, marked as answer]
Are you saying you *are* an Adobe employee?

For external contributions and discussions, we've mostly kept them to Github discussions and issues.
